Source/ACE/Entity/Player.cs

```diff
@@ -29,6 +29,8 @@ public sealed class Player : Creature, IPlayer
         private static readonly Position MarketplaceDrop = new Position(23855548, 49.206f, -31.935f, 0.005f, 0f, 0f, -0.7071068f, 0.7071068f); // PCAP verified drop
         private static readonly float PickUpDistance = .75f;
 
+        private uint coinValue = 100000;
```

Comment:
rather than leaving something hacky like this, use something a little more robust like an admin/dev command for it.  /givepyreals 10000